app/communities-configs/filecoin/images/filecoin-logo.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you have a change to optimize it via svg optimizers like this one?
https://jakearchibald.github.io/svgomg/
<img width="219" alt="image" src="https://user-images.githubusercontent.com/28437795/159867396-74dd2207-f4d8-4972-b5eb-abd245b783d1.png">
I see that we are able to reduce size of icon a little